### DISCUSSION TITLE
Not able to type in UART1 terminal

### DISCUSSION THREAD

**Discussion by Shivaraj199** [polarfire-soc/polarfire-soc-documentation#246] [General] [answered]
Hello guys, 
I am facing this issue since this morning. I am able to flash the LINUX image generated using Yocto. I  could see the serial port printing HSS boot in UART0. After the image is flashed , I can enter the command boot and could see Uboot and LINUX loading in UART1 serial console.
The issue is am not able to type commands inside the serial console UART1. It was working fine till yesterday. I am facing this issue since since today.I tried flashing the prebuilt image too but the problem persist.
Basically i am not even able to enter command root at the login

Baudrate is 115200 and no hardware flow control, no parity.
Can someone tell my why I am having this issue?
Thanks

**Comment by Shivaraj199** [comment, marked as answer]
Hello guys, okay I tried with another serial emulator and it works. Its bit weird to me as  I was using minicom since the beginning and it never threw me this type of problem
